libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -15,4 +15,12 @@ export class ReadingListComponent {
   removeFromReadingList(item) {
     this.store.dispatch(removeFromReadingList({ item }));
   }
+
+  markBookFinished(item) {
+    this.store.dispatch(markBookFinished({ item: { ...item, finishedDate: new Date().toISOString() } }));
+  }
+
+  markBookNotFinished(item) {
+    this.store.dispatch(markBookNotFinished({ item: { ...item, finished: false, finishedDate: undefined } }));
```

Comment:
You know what the values of `finished` and `finishedDate` will be when handling this action. Do you need to provide them? It wouldn't make sense for this action to contain any other values, so why allow that to happen?